### DISCUSSION TITLE
Bot not starting

### DISCUSSION THREAD

**Discussion by nullbyto** [jagrosh/MusicBot#1216] [Troubleshooting]
Bot is not starting as usual, it loads the config and then it doesn't output anything and doesn't go online.
I have already turned on the intents in the developer portal.

I have even created another app from the developer portal and added the example config and only changed prefix owner and token. 
Output:
```
[22:47:59] [INFO] [JMusicBot]: Loaded config from /root/other/MusicBot/config.txt
[22:47:59] [WARN] [Settings]: Failed to load server settings (this is normal if no settings have been set yet): java.nio.file.NoSuchFileException: serversettings.json
```

**Comment by FrankieLu137** [comment]
the same question

**Comment by GRTUBORG** [comment]
what's in your file `serversettings.json` and `config.txt`?

**Reply by nullbyto** [reply]
I don't have `serversettings.json` and in `config.txt` i took the template from https://jmusicbot.com/config/ , added my bot token, prefix and my user id, that's it.
